modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLManagementToolbarPropsTransformer.js

```diff
@@ -231,6 +232,37 @@ export default function propsTransformer({
 		});
 	};
 
+	const permissions = () => {
+		const searchContainer = Liferay.SearchContainer.get(
+			otherProps.searchContainerId
+		);
+
+		if (searchContainer.select) {
+			const keys = searchContainer.select
+				.getAllSelectedElements()
+				.get('value');
+
+			if (keys) {
+				const url = new URL(permissionsURL);
+
+				const urlSearchParams = new URLSearchParams(url.search);
+
+				const paramName = `_${urlSearchParams.get(
+					'p_p_id'
+				)}_resourcePrimKey`;
+
+				for (const key of keys) {
+					url.searchParams.append(paramName, key);
+				}
```

Comment:
This looks a bit complex. If I understood correctly, we are doing this to get first the "paramName" which is set in the searchContainer. But it should be something fixed, so we can get it at the first render of the component, instead of whenever the permissions is clicked. Or better if we pass this value to the component from the JSP...¿?

Reply:
I think that yes?...

I followed the pattern of ManagementToolbarPropsTransformer.js https://github.com/liferay/liferay-portal/blob/9ba17e6a6ff1727924a2b59d3fabd2f8cbcbb123/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ManagementToolbarPropsTransformer.js#L71 as @boton suggest but I'm totally lost at this part

Reply:
Yep, we did it last summer for translation. 

I suggested to @adolfopa to use the addParam from frontend-web but sadly this only updates the param and we need to add the same number of params as selected items.

Reply:
> Or better if we pass this value to the component from the JSP...¿?

We don't know at this point the searchContianer is rendered without selection later the user selects items and launches an action and this action is `permissions` in this case.